.github/workflows/build.yml

```diff
@@ -119,6 +122,11 @@ jobs:
           CXX: "${{ matrix.cxx }}"
           TRAVIS_OS_NAME: "${{ matrix.os }}"
 
+      - name: Create App Bundle
+        run: |
+          meson compile -C build macos-bundle
+          tar -czvf mpv.tar.gz -C build mpv.app
```

Comment:
Why tar it?

Reply:
to preserve file permissions. actions/upload-artifact@v4 action can only zip files and you will lose file permissions. 
see the readme https://github.com/actions/upload-artifact?tab=readme-ov-file#permission-loss and their suggested solution.

Reply:
Ok, clear.